gaudi/gaudi_nfd_instance_openshift.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Adapted from https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/Environment_Setup.html#id2
+#
+apiVersion: nfd.openshift.io/v1
+kind: NodeFeatureDiscovery
+metadata:
+  name: nfd-instance
+  namespace: openshift-nfd
+spec:
+  customConfig:
+    configData: |
+      #    - name: "more.kernel.features"
```

Comment:
can we remove the comments here?

Reply:
It is taken from the Habana documentation. I suggest to keep it the same.